Home page updates (#528)
rudderlabs/rudderstack-docs: user-guides/migration-guides/home.mdx

---
#slug: "/docs"
title: "Home"
description: Documentation for RudderStack - the Customer Data Platform for Developers.
---

# Home

[RudderStack](https://rudderstack.com/) is a customer data platform for developers. Our tooling makes it easy to deploy pipelines that collect customer data from every app, website and SaaS platform, then activate it in your warehouse and business tools.

<div class="successBlock">

> Start building a better, warehouse-first CDP that delivers complete, unified data to every part of your customer data stack. Sign up for <a href="https://app.rudderstack.com/signup?type=freetrial" target="_blank">RudderStack Cloud</a> today.

Reviewer: ```suggestion
Start building a better, warehouse-first CDP that delivers complete, unified data to every part of your customer data stack. Get quickly started by signing up for <a href="https://app.rudderstack.com/signup?type=freetrial" target="_blank">RudderStack Cloud</a>.
```

</div>

## Product onboarding

<details>
<summary><b><a href="https://www.rudderstack.com/docs/rudderstack-cloud/">RudderStack Cloud</a></b></summary>

> Familiarize yourself with RudderStack Cloud and the different dashboard options available to you once you sign up.

Reviewer: ```suggestion
Familiarize yourself with RudderStack Cloud and the different dashboard options once you sign up.
```

</details>

<details>
<summary><b><a href="https://www.rudderstack.com/docs/rudderstack-open-source/">RudderStack Open Source</a></b></summary>

Follow the guides listed in this section to set up RudderStack in your own environment.

</details>

<details>

> <summary><b><a href="https://www.rudderstack.com/docs/user-guides/migration-guides/">Glossary</a></b></summary>

Author: ```suggestion
<summary><b><a href="https://www.rudderstack.com/docs/get-started/glossary/">Glossary</a></b></summary>
```

> Familiarize yourself with RudderStack and the commonly occurring terms.

Reviewer: ```suggestion
Familiarize yourself with RudderStack and the commonly associated terms with it.
```

</details>

## Quick start guides

<details>
<summary><b><a href="https://www.rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/">SDK Guides</a></b></summary>

Integrate RudderStack with your web, mobile, and server-side apps and start tracking event data in no time.

</details>

<details>
<summary><b><a href="https://www.rudderstack.com/docs/destinations/">Event Stream Destinations</a></b></summary>

Integrate RudderStack with over 150 third-party tools/platforms.

</details>

<details>
<summary><b><a href="https://www.rudderstack.com/docs/data-warehouse-integrations/">Warehouse Destinations</a></b></summary>

Integrate RudderStack with your preferred data warehouse and make it the single source of truth for all your customer data.

</details>

<details>
<summary><b><a href="https://www.rudderstack.com/docs/cloud-extract-sources/">Cloud Extract</a></b></summary>

> Integrate RudderStack with your preferred data warehouse and make it the single source of truth for all your customer data.

Reviewer: ```suggestion
Integrate RudderStack with your preferred cloud tools and make it the single source of truth for all your customer data.
```

</details>

<details>
<summary><b><a href="https://www.rudderstack.com/docs/reverse-etl/">Reverse ETL</a></b></summary>

> Leverage the enriched customer data residing in your data warehouse and route it to your preferred downstream destinations.

Reviewer: ```suggestion
Leverage the customer data residing in your data warehouse and route it to your preferred downstream destinations.
```

</details>

> ## Features deep-dive

Reviewer: ```suggestion
## Useful features
```

<details>
<summary><b><a href="https://www.rudderstack.com/docs/transformations/">Transformations</a></b></summary>

 Code custom JavaScript functions to transform your event data before sending it to your destinations.

</details>

<details>
<summary><b><a href="https://www.rudderstack.com/docs/data-governance/">Data Governance</a></b></summary>

Programmatically access your events and their metadata and fix any inconsistencies.

</details>

<details>
<summary><b><a href="https://www.rudderstack.com/docs/rudderstack-api/">RudderStack API</a></b></summary>

Get detailed instructions on using various RudderStack API based on your use-cases.

</details>

## Need help?

<details>
<summary><b><a href="https://www.rudderstack.com/docs/user-guides/migration-guides/">Migration Guides</a></b></summary>

> Migrating from Segment? Refer to these guides for help.

Reviewer: ```suggestion
Migrating from Segment or Blendo? Refer to these guides for help.
```

</details>

<details>

> <summary><b><a href="https://www.rudderstack.com/docs/user-guides/help-guides/">Support Guides</a></b></summary>

Reviewer: ```suggestion
<summary><b><a href="https://www.rudderstack.com/docs/user-guides/how-to-guides/">Support Guides</a></b></summary>
```
